back/src/dals/session/repository/session.repository.ts

```diff
@@ -0,0 +1,74 @@
+import { InputMessageTypes } from 'messages';
+import {ConnectSessionInfo, RoomInfo, UserSession, UserSessionContext, RoomContext} from 'dals'
+
+export const isRoomAvailable = async (room: string): Promise<boolean> => {
```

Comment:
Why don't you have this room functionality in `dals/room/room.repository`?

Reply:
I started to do so but I had to rollback since I was not completely sure how to solve this (dals/session/repository/index.ts):

import * as mockRepository from './session.mock';
import * as repository from './session.repository';
export const sessionRepository: SessionRepository = envConstants.isMockRepository
  ? mockRepository
  : repository;

Reply:
We should have two dals `dals/session` and `dals/room` with its own repository.

Then, you can import like:
```
import { sessionRepository } from 'dals/session';
import { roomRepository } from 'dals/room';
```

Reply:
Done, dals have been spplited in two entities: roomRepository and sessionRepository